### ISSUE TITLE
CI locks?

### ISSUE THREAD

**Issue by orta** [artsy/eigen#985]
It looks like it stopped here: https://circle-artifacts.com/gh/artsy/eigen/1078/artifacts/0/tmp/circle-artifacts.KgPk57x/xcode_test_raw.log

```
Test Case '-[ARTopMenuNavigationDataSourceSpec test_notifications__resets_the_app_icon_badge_if_there_are_0_notifications]' started.
2015-12-14 14:43:05.928 Artsy[6054:11667] Rewriting /shows as https://m.artsy.net/shows
```

**Comment by ashfurrow** [comment]
Saw this today, can replicate it locally.

**Comment by ashfurrow** [comment]
See this still, will put some time into it when I get a chance.

**Comment by orta** [comment]
fixed by #1068
